docs/node-operators/hot-cold-block-production.mdx

```diff
@@ -1,58 +1,80 @@
+Next, ensure that you have delegated your cold wallet account to your hot wallet account. 
+
+- If you've set up your account to delegate to the correct spot in the genesis ledger, then no further action is needed. 
+- If not, you can delegate by using the `mina client delegate-stake` command if your key was generated on a computer. Otherwise, follow the delegate instructions on the [Mina ledger README](https://github.com/jspada/ledger-app-mina/tree/v1.0.0-beta.2).
 
 ## Block Production
 
-Follow the "Start up a node" instructions in the [Connecting to the Network](connecting-to-the-network) page using your hot wallet key.
+Follow the start up a node instructions in [Connecting to the Network](connecting-to-the-network) using your hot wallet key.
 
 ## Protocol Details
 
 You may be thinking: Why can't I create blocks directly from a secure enclave or other HSM device in Mina's protocol?
 
-Two components of block production require use of the private key in interesting ways: Winning a slot and creating the Blockchain SNARK. Creating the SNARK within the secure hardware especially is extremely difficult and likely impossible to do quickly enough for our protocol given today's technology.
+Two components of block production require using a private key in interesting ways: 
+
+- Winning a slot
+- Creating the blockchain SNARK
+
+Creating the SNARK within the secure hardware is extremely difficult and likely impossible to do quickly enough for the protocol with today's technology.
+
+### Winning a slot (out of the SNARK)
 
-### Winning a slot (out of the snark)
+A block producer can determine if they've won a slot by evaluating a VRF ( verifiable random function) using their private key. The VRF needs to be evaluated for your account and every account that delegates to you for all slots within an epoch. To our knowledge, no one has ever implemented a VRF on an HSM, but this is likely not too difficult.
 
-A block producer can determine if they've won a slot by evaluating a VRF (or verifiable random function) using their private key. The VRF needs to be evaluated for your account and every account that delegates to you for all slots within an epoch. To our knowledge, no one has ever implemented a VRF on an HSM, but this is likely not too difficult.
+### Creating the blockchain SNARK
 
-### Creating the Blockchain SNARK
+When a block producer wins a slot, in addition to just creating and broadcasting a block, a block producer must create a new SNARK proof that proves that the blockchain is valid including this new block. 
```

Comment:
Recommend rephrasing: "proves that the new block is a valid extension of the existing chain"

Reply:
such great feedback, thank you @nholland94  

Do you know about the GitHub suggesting feature? Love it for lightweight in-line suggestions like this 

I wish that GH had a more succinct doc for my favorite feature 
see step 4 https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request